config/prod/prow/config_knative.yaml

```diff
@@ -277,6 +277,11 @@ presubmits:
     always-run: true
     command:
     - ./test/presubmit-integration-tests-latest-release.sh
+  knative/client-pkg:
+  - build-tests: true
+  - unit-tests: true
+  - integration-tests: true
+  - go-coverage: false
```

Comment:
no coverage ?

Reply:
We'll have still have Codecov and only skip the go-coverage from Prow.

E.g.: https://github.com/knative/test-infra/blob/main/config/prod/prow/config_knative.yaml#L24